dist/main.js

```diff
@@ -1,2 +1,208 @@
\ No newline at end of file
+/*
+ * ATTENTION: The "eval" devtool has been used (maybe by default in mode: "development").
+ * This devtool is neither made for production nor for readable output files.
+ * It uses "eval()" calls to create a separate source file in the browser devtools.
+ * If you are trying to read the output file, select a different devtool (https://webpack.js.org/configuration/devtool/)
+ * or disable the default devtool with "devtool: false".
+ * If you are looking for production-ready output files, see mode: "production" (https://webpack.js.org/configuration/mode/).
+ */
+/******/ (() => { // webpackBootstrap
+/******/ 	"use strict";
+/******/ 	var __webpack_modules__ = ({
+
+/***/ "./node_modules/css-loader/dist/cjs.js!./src/style.css":
+/*!*************************************************************!*\
+  !*** ./node_modules/css-loader/dist/cjs.js!./src/style.css ***!
```

Comment:
One of the best practices for JavaScript is to add node_modules dir to your `.gitignore` file as all those files are not needed in your repo (each team member will install all packages thanks to your package.json file). 
Therefore, kindly delete the `node_modules` directory, add it to the `.gitignore` file, and commit your changes.